tests/integration/test_matrix_power.py

```diff
@@ -73,28 +73,39 @@ class TestMatrixPowerErrors:
     def test_matrix_ndim_smaller_than_two(self, ndim):
         shape = (3,) * ndim
         a_num = mk_0to1_array(num, shape)
-        msg = "Expected at least 2d array"
-        with pytest.raises(num.linalg.LinAlgError, match=msg):
+        a_np = mk_0to1_array(np, shape)
+        expected_exc = num.linalg.LinAlgError
+        expected_exc_np = np.linalg.LinAlgError
+        with pytest.raises(expected_exc):
             num.linalg.matrix_power(a_num, 1)
+        with pytest.raises(expected_exc_np):
+            np.linalg.matrix_power(a_np, 1)
 
     @pytest.mark.parametrize(
         "shape", ((2, 1), (2, 2, 1)), ids=lambda shape: f"(shape={shape})"
     )
     def test_matrix_not_square(self, shape):
         a_num = mk_0to1_array(num, shape)
-        msg = "Last 2 dimensions of the array must be square"
-        with pytest.raises(num.linalg.LinAlgError, match=msg):
+        a_np = mk_0to1_array(np, shape)
+        expected_exc = num.linalg.LinAlgError
+        expected_exc_np = np.linalg.LinAlgError
```

Comment:
@manopapad is there a reason we define our own version of `LinAlgError` instead of using `np.linalg.LinAlgError`.

Reply:
I don't think so. We need to provide a definition for `cunumeric.linalg.LinAlgError`, so that replacing `import numpy as X` with `import cunumeric as X` works out of the box, but it should be fine to just alias that to `np.linalg.LinAlgError`.

Reply:
Should I keep the codes here or do some changes in the definition of `cunumeric.linalg.LinAlgError`?

Reply:
@XiaLuNV Can you change [`cunumeric.linalg.exception`](https://github.com/nv-legate/cunumeric/blob/branch-23.09/cunumeric/linalg/exception.py) to simply import and expose the existing Numpy exception, rather than defining a new version? Then this test can look like the other tests with just a single "expected" value (having to have two values really defeats the point of the test)